services/user/Sites/src/Sites.cpp

```diff
@@ -6,13 +6,41 @@
 #include <psibase/servePackAction.hpp>
 #include <psibase/serveSchema.hpp>
 #include <psibase/serveSimpleUI.hpp>
+#include <regex>
+#include <services/system/Accounts.hpp>
+#include <sstream>
 
 using namespace psibase;
 
 namespace SystemService
 {
    namespace
    {
+      std::string to_lower(const std::string& str)
+      {
+         std::string lower;
+         std::transform(str.begin(), str.end(), std::back_inserter(lower), ::tolower);
+         return lower;
+      }
+
+      template <typename T>
+      bool contains(const std::vector<T>& vec, const T& value)
```

Comment:
I've been considering bumping to c++23, which has `std::ranges::contains`. What do you think?

Reply:
+1 would be nice